src/gsi/gsi_dbzOper.F90

```diff
@@ -99,8 +103,30 @@ subroutine setup_(self, lunin, mype, is, nobs, init_pass,last_pass)
     character(len=len_isis   ):: isis
     integer(i_kind):: nreal,nchanl,ier,nele
     logical:: diagsave
+    integer(i_kind):: lu_diag
+    character(128):: diag_file
+    character(80):: string
+
+    if(nobs == 0) then
+
+       if(mype == 0) then
+          write(6,*) 'init_pass = ', init_pass
+          write(6,*) 'l_use_dbz_directDA = ', l_use_dbz_directDA
+       endif
 
-    if(nobs == 0) return
```

Comment:
Lines 117-125 are for making sure that diag files start with the date.  This solution is messy, but it works.